textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/regexmatch/RegexToGramQueryTranslatorTest.java

```diff
@@ -0,0 +1,29 @@
+package edu.uci.ics.textdb.dataflow.regexmatch;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * @author Shuying Lai
+ * @author Zuozhi Wang
+ */
+
+public class RegexToGramQueryTranslatorTest {
```

Comment:
For testing translator, because we only implemented empty string and the star expression, so we can only test these two. More test cases will be added as we finished implementing a new expression.

For `equals`, I'll add more test cases.